dspace/config/modules/storage.cfg

```diff
@@ -0,0 +1,31 @@
+#---------------------------------------------------------------#
+#-----------------STORAGE CONFIGURATIONS------------------------#
+#---------------------------------------------------------------#
+# Configuration properties used by the bitstore.xml config file #
+#                                                               #
+#---------------------------------------------------------------#
+
+# Use the localStore or the s3Store implementation
+assetstore.storename.0 = localStore
+
+# For using a relative path (xx/xx/xx/xxx...) set to true, default it false
+assetstore.s3.useRelativePath = false
```

Comment:
I'm not sure I understand this configuration. Could we describe it more clearly?  Where does this "relativePath" come from?  Can we give an example in the comments?